_template/chain.json

```diff
@@ -0,0 +1,105 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "",
+  "status": "live",
+  "website": "",
+  "network_type": "mainnet",
+  "pretty_name": "",
+  "chain_id": "",
+  "bech32_prefix": "",
+  "daemon_name": "",
+  "node_home": "$HOME/",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0,
+        "average_gas_price": 0.025,
+        "high_gas_price": 0.04
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": ""
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "",
+    "recommended_version": "v1.0",
+    "compatible_versions": [
+      "v1"
```

Comment:
We should probably make clear whether or not recommended version must be listed under compatible versions as well. Since it seems to be the more common, I'd say so, and therefore we should write 'v1.0' here (and not v1)

Reply:
oh, actually, seeing the use of v1 below, it's the recommended_version that should change to 'v1'